client/js/nug-skeleton-app.js

```diff
@@ -1,24 +1,109 @@
   'use strict';
 
   var nugNgApp = angular.module('nug-skeleton-app', [
-    'ui.router'
+    'ui.router','auth0', 'angular-storage', 'angular-jwt'
   ]);
 
-  nugNgApp.config(function($stateProvider, $urlRouterProvider) {
+  nugNgApp.config(function($stateProvider, $urlRouterProvider, authProvider, $httpProvider, jwtInterceptorProvider) {
     $stateProvider
       .state('dashboard', {
         url: '/dashboard',
         templateUrl: 'partials/dashboard.html',
-        controller: 'dashboardCtrl'
+        controller: 'dashboardCtrl',
+        data: {
+     		requiresLogin: true
+   		}
       })
       .state('settings', {
         url: '/settings',
         templateUrl: 'partials/settings.html',
-        controller: 'settingsCtrl'
-      });
-  });
+        controller: 'settingsCtrl',
+         data: {
+     		requiresLogin: true
+   		}
+      })
+	  .state('login', {
+		url: '/login',
+		templateUrl: 'partials/login.html',
+		controller: 'loginCtrl'
+	  })
+	;
+  
+  	//used for Auth0
+  
+		authProvider.init({
+		domain: 'nugs.auth0.com',
+		clientID: 'TdtdYCDQHSR3TtNgsMuCXHfjHDyxMsmB',
+		callbackURL: location.href,
+    	// Here include the URL to redirect to if the user tries to access a resource when not authenticated.
+    	loginUrl: '/login',
+    	loginState: 'login'
+    	
+		});
+		
+	authProvider.on('loginSuccess', function($location, profilePromise, idToken, store) {
+	  console.log("Login Success");
+	  profilePromise.then(function(profile) {
+		store.set('profile', profile);
+	//	console.log(profile);
+	  
+		store.set('token', idToken);
```

Comment:
If we're using local storage for the auth token, does that mean I'm not authenticated across devices?  Like, if I login from my mobile, do I need to re-login from my desktop?  Maybe that doesn't matter?